server/liveblog/themes/themes_assets/default/misc/filters.js

```diff
@@ -100,6 +100,15 @@ const decodeUri = (url) => {
   return decodeURIComponent(url);
 }
 
+/**
+ * Converts a JavaScript object to a JSON string.
+ * @param  {Object} obj The object to be converted to JSON.
+ * @return {string} JSON string representation of the object.
+ */
+const tojson = (obj) => {
+  return JSON.stringify(obj);
+}
+
```

Comment:
Filters need a counter part in the Python side, otherwise the embed generation will fail. See here

https://github.com/liveblog/liveblog/blob/master/server/liveblog/themes/template/filters.py
https://github.com/liveblog/liveblog/blob/master/server/liveblog/themes/themes.py#L291-L296

Reply:
I didn't add a counterpart filter on the Python side since Jinja already has an inbuilt filter of the same name : https://jinja.palletsprojects.com/en/3.0.x/templates/#jinja-filters.tojson

Reply:
Got it, thanks for clarifying.